README.md

```diff
@@ -17,17 +17,28 @@ instance, includes a
 [Cockpit Module](https://github.com/osbuild/cockpit-composer) or using the
 [composer-cli](https://weldr.io/lorax/composer-cli.html) command-line tool.
 
-### Project
+## Project
 
  * **Website**: <https://www.osbuild.org>
  * **Bug Tracker**: <https://github.com/osbuild/osbuild-composer/issues>
- * **matrix**: https://matrix.to/#/#image-builder:fedoraproject.org
-*  **Mailing List**: image-builder@redhat.com
+ * **Matrix**: https://matrix.to/#/#image-builder:fedoraproject.org
+ * **Mailing List**: image-builder@redhat.com
```

Comment:
Is this the right list to direct people to? The last message I have is from April 2023, and according to the headers it is a private list (the archives are private at least).

Reply:
It is. There's just almost no traffic there since we don't use it internally and there's no community.